src/ir/module-utils.cpp

```diff
@@ -38,23 +38,40 @@ static void updateLocationSet(std::set<Function::DebugLocation>& locations,
   std::swap(locations, updatedLocations);
 }
 
+static void updateSymbolSet(std::set<Function::DebugLocation>& locations,
+                            std::vector<Index>& symbolIndexMap) {
+  std::set<Function::DebugLocation> updatedLocations;
+
+  for (auto iter : locations) {
+    if (iter.symbolNameIndex.has_value()) {
```

Comment:
```suggestion
    if (iter.symbolNameIndex) {
```
Just checking the boolean value is enough.